src/language-server/utilities/__tests__/source.test.ts

```diff
@@ -0,0 +1,115 @@
+import { extractGraphQLSources } from "../../document";
+import { TextDocument } from "vscode-languageserver-textdocument";
+import {
+  findContainedSourceAndPosition,
+  positionFromPositionInContainingDocument,
+} from "../source";
+import { handleFilePartUpdates } from "../../project/rover/DocumentSynchronization";
+const testText = `import gql from "graphql-tag";
```

Comment:
```suggestion
import { handleFilePartUpdates } from "../../project/rover/DocumentSynchronization";

const testText = `import gql from "graphql-tag";
```

Minor thing, but my eyes initially thought the test text was part of the set of import statements above 😆